### ISSUE TITLE
[Bug] LED does not report a color mode, will stop working in HA Core 2025.3

### ISSUE THREAD

**Issue by sleveille4** [Vaskivskyi/ha-asusrouter#807]
### The problem

since latest core update, a warning appears about LED not reporting a color mode

### Your device model

RT-AX86U

### Firmware type

Stock

### Firmware version

3.0.0.4.388.24231_gbc11d13

### Integration version with the issue

0.29.0

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

core 2024.3.0

### Is there anything useful in the logs?

```txt
Logger: homeassistant.components.light
Source: components/light/__init__.py:914
integration: Light (documentation, issues)
First occurred: 11:07:42 (1 occurrences)
Last logged: 11:07:42

light.rt_ax86u_led (<class 'custom_components.asusrouter.light.ARLightLED'>) does not report a color mode, this will stop working in Home Assistant Core 2025.3, please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
```

**Comment by ellisjm74** [comment]
Seeing the same issue after upgrading to 2024.3

**Comment by ClockwiseCounterclockwise** [comment]
RT-AX88U Pro on Merlin [3004.388.6_2] same issue.

**Comment by Vaskivskyi** [comment]
Hello, guys. Thanks for your reports. This issue will be fixed in #808 and the release will be available in the next 48 hours.

Unfortunately, lately, I haven't had any free time, so I didn't yet know about this new feature of 2024.3

**Comment by Vaskivskyi** [comment]
The release is already available in HACS. In case if the issue persists, please, reopen this issue
